### ISSUE TITLE
fsck errors in system journal

### ISSUE THREAD

**Issue by fasaxc** [coreos/bugs#1257]
I'm not sure if these are benign but, at boot, I noticed fsck is complaining that it doesn't support the filesystem on my coreos host:

```
Apr 29 08:07:18 localhost systemd-fsck[656]: /dev/sda3 has unsupported feature(s): FEATURE_R24 FEATURE_R25 FEATURE_R26 FEATURE_R27 FEATURE_R28 FEATURE_R29 FEATURE_R30 FEATURE_R31
Apr 29 08:07:18 localhost systemd-fsck[656]: e2fsck: Get a newer version of e2fsck!
Apr 29 08:07:18 localhost systemd-fsck[656]: fsck failed with error code 8.
Apr 29 08:07:18 localhost systemd-fsck[656]: Ignoring error.
```

I'm running coreos-alpha-1010-1-0-v20160407 on GCE.

**Comment by crawford** [comment]
This is due to us clobbering the filesystem flags. We do this to try to prevent people from remounting `/usr` read-write. This is safe to ignore.

**Comment by dnorthup-ums** [comment]
@crawford The correct way to fix this is not to ignore errors one finds inconvenient. Please work with the CoreOS core contributors to fix the appropriate systemd unit to not fsck `/usr` or start using the fstype flags appropriately.

**Comment by bgilbert** [comment]
`/usr` is immutable, so it doesn't make sense to fsck it.  On systems with a sufficiently recent bootloader, dm-verity will notice any filesystem corruption that does occur.

https://github.com/coreos/bootengine/pull/126 disables fsck for the `/usr` partition.  That change should be included in 1492.0.0.
